string[] paths
string[] constrained_symbols
---
kineverse/Model model